os/net/linkaddr.h

```diff
@@ -110,6 +110,12 @@ int linkaddr_cmp(const linkaddr_t *addr1, const linkaddr_t *addr2);
  */
 void linkaddr_set_node_addr(linkaddr_t *addr);
 
+/**
+ * \brief      Derives an EUI-64 address from the given address
+ */
+void linkaddr_to_eui_64(uint8_t *dst, const linkaddr_t *addr);
+void linkaddr_from_eui_64(linkaddr_t *addr, const uint8_t *src);
```

Comment:
didn't look very closely, but shouldn't these function use (or be used by) `uip_ds6_set_addr_iid` and `uip_ds6_set_lladdr_from_iid`?

Reply:
Right. Anyway, the actual reason why I moved this conversion to `linkaddr.ch` has gone. I am now doing this conversion inside `uip-ds6.c`.